.openapi-generator/FILES

```diff
@@ -30,21 +31,24 @@ docs/ObjectApi.md
 docs/Operation.md
 docs/PayloadConfig.md
 docs/QUEUESETUPSTATUS.md
-docs/ResponseList.md
+docs/ResponseListObject.md
 docs/SIGN.md
 docs/STORETYPE.md
 docs/Store.md
 docs/SubscriptionApi.md
-docs/SubscriptionConfig.md
+docs/SubscriptionConfigInput.md
```

Comment:
Don't really like that Pydantic V2 creates `ModelName-Input` and `ModelName-Output schemas` just because there are subtle differences like certain fields are optional (because we have defaults for them) but it is like it is